src/containers/Transactions/index.jsx

```diff
@@ -44,23 +44,24 @@ const Transaction = (props) => {
   const { t } = useTranslation()
   const { actions, data, loading } = props
   const rippledSocket = useContext(SocketContext)
+  const hash = data.raw ? data.raw.hash : undefined
+  const short = identifier.substr(0, 8)
+  document.title = `${t('xrpl_explorer')} | ${t(
+    'transaction_short',
+  )} ${short}...`
 
   useEffect(() => {
-    const hash = data.raw ? data.raw.hash : undefined
-    const short = identifier.substr(0, 8)
-
-    document.title = `${t('xrpl_explorer')} | ${t(
```

Comment:
We should still update title.

Reply:
It's updated outside of the `useEffect` now (line 49). Should it stay in the useEffect?

Reply:
Sorry I missed that.  It is fine outside of useEffect.  Just check when switching to a different TX it still updates

Reply:
Confirmed that it does